Makefile

```diff
@@ -141,6 +143,7 @@ yt-dlp.tar.gz: all
 		--exclude '__pycache__' \
 		--exclude '.pytest_cache' \
 		--exclude '.git' \
+		--exclude '__pyinstaller' \
```

Comment:
Pretty sure this is wrong

Reply:
it was, fixed in 43cfd462c0d01eff22c1d4290aeb96eb1ea2c0e1